projects/plugins/boost/app/modules/optimizations/lazy-images/class-lazy-images.php

```diff
@@ -16,6 +16,13 @@ public static function get_slug() {
 	}
 
 	public static function is_available() {
+		// In some scenarios, Lazy Images can conflict with other WordPress features.
+		// If we detect that, the feature should not be available.
+		if ( Jetpack_Lazy_Images::should_force_deactivate() ) {
+			Jetpack_Lazy_Images::disable();
```

Comment:
My only recommendation is to avoid disabling the module here. This function is for checking the availability